board/post-build.sh

```diff
@@ -33,3 +33,15 @@ ln -sf /dev/null "${TARGET_DIR}"/etc/systemd/system/systemd-update-utmp-runlevel
 if [[ ${TARGET_DIR} != *"raspberrypi0cam"* ]]; then
   ln -sf /dev/null "${TARGET_DIR}"/etc/systemd/system/network.service
 fi
+
+# Set os-release file
+SHOWMEWEBCAM_VERSION=$(support/scripts/setlocalversion "${BR2_EXTERNAL_PICAM_PATH}")
+
+cat > "${TARGET_DIR}"/usr/lib/os-release <<EOF
+NAME="Show-me webcam"
+VERSION=${SHOWMEWEBCAM_VERSION}
+ID=showmewebcam
+VERSION_ID=${SHOWMEWEBCAM_VERSION}
+BUILD_ID=$(date +%Y.%m.%d)-${SHOWMEWEBCAM_VERSION}
```

Comment:
I'm a fan of [reproducible builds](https://wiki.debian.org/ReproducibleBuilds).
How about using the commit date and time for `HEAD` instead?

Reply:
You are absolutely right. However it's quite hard to get the actual commit date at that time. So I've just removed that line as it is optional anyway and we've got the same information in `VERSION_ID` already.